packages/genui/playground/src/pages/BenchPage.css

```
@@ -9,13 +9,26 @@
     var(--geist-background) 280px
   );
 }
 
+.benchPageAccessibleTitle {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)